packages/digit_dss/lib/models/entities/dashboard_request.dart

```diff
@@ -0,0 +1,56 @@
+// Generated using mason. Do not modify by hand
+import 'package:dart_mappable/dart_mappable.dart';
+
+part 'dashboard_request.mapper.dart';
+
+@MappableClass(ignoreNull: true, discriminatorValue: MappableClass.useAsDefault)
+class DashboardRequestModel with DashboardRequestModelMappable {
+  final AggregationRequestDto aggregationRequestDto;
+  final DSSHeaders headers;
+
+  DashboardRequestModel({
+    required this.aggregationRequestDto,
+    required this.headers,
+  }) : super();
+}
+
+@MappableClass(ignoreNull: true, discriminatorValue: MappableClass.useAsDefault)
+class AggregationRequestDto with AggregationRequestDtoMappable {
+  final String visualizationCode;
+  final String visualizationType;
+  final Map? filters;
```

Comment:
See if you can give Map<> type

Reply:
Filters we are not using for mobile dashboard, As of now, it need to be empty map object, If needed in future, we will add this,